researcher_ids/RI_Claims_V1.md

```diff
@@ -279,41 +393,162 @@ interfaces, such as:
 -   May provide a rich set of internationalization/localization features for
     clients to consume.
 
-## Claim Objects
+## Passport JWT Claim Format
 
-Each [RI claim](#researcher-identity-claim-ri-claim) name maps to an array of
-claim JSON objects ([RI Claim Objects](#researcher-identity-claim-object-ri-claim-object))
-within a "ga4gh" root OIDC claim object (see [example](#example-ri-claims)).
+The [Passport JWT Claim](#passport-jwt-claim) name maps to an array of
+[Passport Visas](#passport-visa) which are encoded as
+[Embedded Tokens](https://github.com/ga4gh/data-security/blob/master/AAI/AAIConnectProfile.md#term-embedded-token)
+within a Passport JWT Claim.
 
-## Claim Object Fields
+Non-normative example of a set of Passport Visas, encoded as Embedded
+Token JWS Compact Serialization strings:
 
-Fields within a RI Claim Object are:
+```
+{
+  "ga4gh_passport_v1": [
+    "<eyJhbGciOiJI...aaa>",
+    "<eyJhbGciOiJI...bbb>"
+  ]
+}
+```
+
+For a more reader-friendly example, see the [Example Passport JWT
+Claim](#example-passport-jwt-claim) section of the specification.
+
+### Passport Visa Requirements
 
-#### "**value**" [required]
+-   Passport Visas MUST conform to one of the
+    [GA4GH AAI Specification Embedded Token formats](https://github.com/ga4gh/data-security/blob/master/AAI/AAIConnectProfile.md#embedded-token-issued-by-embedded-token-signatories)
+    as JWS Compact Serialization string as defined by [RFC7515 section
+    7.1](https://tools.ietf.org/html/rfc7515#section-7.1).
 
--   A string that represents the type, process and version of the claim. The
-    format of the string can vary by the
-    [claim definition](#ga4gh-researcher-identity-claim-definitions).
+-   Embedded Token Signatories, Passport Brokers, Passport Clearinghouses, and
+    their Clients MUST conform to the
```

Comment:
"and their clients" Where does the scope of this spec end? I don't think we should impose requirements to actors who are not covered by this document

Reply:
Limited the scope of client conformance to just the part that the AAI spec calls out.